module/x/gravity/abci.go

```diff
@@ -120,6 +121,18 @@ func eventVoteRecordTally(ctx sdk.Context, k keeper.Keeper) {
 	// Then we sort it
 	sort.Slice(keys, func(i, j int) bool { return keys[i] < keys[j] })
 
+	// we delete all attestations earlier than the current event nonce
+	// minus some buffer value. This buffer value is purely to allow
+	// frontends and other UI components to view recent oracle history
+	const eventsToKeep = 1000
```

Comment:
changed to `100`

I am planning to open a PR to parametrised bunch of hardcoded value later on.

Reply:
or just prune all and disable node querying for it (whoever wants to query, they can use an indexer?)?

it seems a bit strange to have a magic "frontend-only" number in consensus-critical code which doesn't do anything about frontend?

Reply:
I created a ticket to parametized it 
https://github.com/crypto-org-chain/gravity-bridge/issues/82


I guess this could be usefull to detect who are not sending the attestation without directly relying on an indexer. But this could be set to 0 also.

Unfortunately we cannot make it a node params since its rely on deleting on the state storage

Reply:
if one doesn't have a state pruning enabled, one can get the same information by querying the state a particular historical height?